.gitignore

```diff
@@ -6,6 +6,7 @@
 /cmake-build-*
 /.vs
 /CMakeSettings.json
+/out

```

Comment:
I added this because in Visual Studio 2019, the default output directory for CMake Projects is "out/build/{buildType}".